src/raft.rs

```diff
@@ -796,7 +773,17 @@ impl<T: Storage> Raft<T> {
             (MessageType::MsgRequestVote, self.term)
         };
         let self_id = self.id;
-        if self.quorum() == self.poll(self_id, vote_resp_msg_type(vote_msg), true) {
+        let acceptance = true;
+        info!(
+            "{} received {:?}{} from {} at term {}",
+            self.id,
+            vote_msg,
+            if !acceptance { " rejection" } else { "" },
```

Comment:
`acceptance` must be true here. So could we simplify it?

Reply:
I suppose so, I wanted to keep it the same as the other log. I'll change it. :)